ProcessMaker/Nayra/Managers/WorkflowManagerRabbitMq.php

```diff
@@ -10,13 +10,15 @@
 use ProcessMaker\Facades\MessageBrokerService;
 use ProcessMaker\Facades\WorkflowManager;
 use ProcessMaker\GenerateAccessToken;
+use ProcessMaker\Jobs\RunNayraServiceTask;
 use ProcessMaker\Managers\DataManager;
 use ProcessMaker\Managers\SignalManager;
 use ProcessMaker\Models\EnvironmentVariable;
-use ProcessMaker\Models\Process as Definitions;
 use ProcessMaker\Models\Process;
+use ProcessMaker\Models\Process as Definitions;
```

Comment:
this reference is not used in this file...

Reply:
Definitions is used as alias of Process. I removed the `use ProcessMaker\Models\Process;` and use `Definitions` to keep the logic of the class